Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsIT.cs

```diff
@@ -115,7 +117,7 @@ public void TestBasicConnectionPool()
         public void TestWaitForTheIdleConnectionWhenExceedingMaxConnectionsLimit()
         {
             // arrange
-            var connectionString = ConnectionString + "application=TestWaitForMaxSize1;waitingForIdleSessionTimeout=1s;maxPoolSize=2";
+            var connectionString = ConnectionString + "application=TestWaitForMaxSize1;waitingForIdleSessionTimeout=1s;maxPoolSize=2;minPoolSize=1";
```

Comment:
unfortunately application name is limited by its length or format so it cannot be just a test name. I introduced application name in a few tests in this class because I had to put some effort to tune the tests not to be flaky so I wanted to make sure other tests cannot interfere with given test.